### PR TITLE
CLI: get the NOQUATTOR constants from CCM::Fetch::Config

### LINKED ISSUE #89: Use NOQUATTOR constants from CCM::Fetch::Config
Avoids ENV polution by CCM::Fetch::Download
Requires https://github.com/quattor/CCM/pull/136

### PR DESCRIPTION
**Pull request by stdweird**
Fixes #89

### CODE REVIEW

**Comment by stdweird** [comment]
@jrha here you go, simple fix

File: src/main/perl/CLI.pm [hunk 1]
```diff
@@ -6,7 +6,7 @@ use CAF::Reporter 16.8.1 qw($LOGFILE);
 use CAF::Lock qw(FORCE_IF_STALE FORCE_ALWAYS);
 use CAF::Object qw (SUCCESS throw_error);
 use EDG::WP4::CCM::CacheManager;
-use EDG::WP4::CCM::Fetch qw(NOQUATTOR NOQUATTOR_EXITCODE NOQUATTOR_FORCE);
+use EDG::WP4::CCM::Fetch::Config 16.8.1 qw(NOQUATTOR NOQUATTOR_EXITCODE NOQUATTOR_FORCE);
```
Comment (jrha) [inline src/main/perl/CLI.pm]: Where does this version number come from?

Comment (stdweird) [inline src/main/perl/CLI.pm]: it's more recent than 16.8.0 release and when you make rpms from the current master, you get 16.8.1-SNAPSHOT....; so this allows you to ask for something more recent than 16.8 and still be able to test it.

Comment (jrha) [inline src/main/perl/CLI.pm]: :+1:

**Pull request merged**